README.md

```diff
@@ -1823,7 +1823,7 @@ Sets the list of resources to look for when a client requests an index of the di
 
 ##### `docroot`
 
-**Required**. Sets the [`DocumentRoot`][] location, from which Apache serves files.
```

Comment:
In Puppet 4, I was expecting Optional[String] $test to be the same as $test = undef. But it is not. So you have to use Optional[String] $test = undef. It means that undef is a thing.